web/php/Modules/Account/PM/PMDraftsModule.php

```diff
@@ -1,55 +1,52 @@
 <?php
 
-namespace Wikidot\Modules\Account\PM;
-
-
+declare(strict_types=1);
 
+namespace Wikidot\Modules\Account\PM;
 
-use Ozone\Framework\Database\Criteria;
-use Wikidot\DB\PrivateMessagePeer;
 use Wikidot\Utils\AccountBaseModule;
+use Wikijump\Models\UserMessage;
 
+/**
+ * AJAX Module for per-page view of drafts.
+ * @package Wikidot\Modules\Account\PM
+ */
 class PMDraftsModule extends AccountBaseModule
 {
 
+    /**
+     * Build drafts view.
+     * @param $runData
+     */
     public function build($runData)
     {
+        /** Not sure if this is worth breaking out into a setting. */
+        $per_page_limit = 30;
 
-        $userId = $runData->getUserId();
-
-        $pl = $runData->getParameterList();
-
-        $pageNo = $pl->getParameterValue("page");
-        if ($pageNo == null || $pageNo<0) {
-            $pageNo = 1;
+        $this_page = $runData->get('page');
+        if ($this_page == null || $this_page < 0) {
+            $this_page = 1;
         }
 
-        // get inbox messages for the user
-        $c = new Criteria();
-        $c->add("from_user_id", $userId);
-        $c->add("flag", 2); // for inbox
-
-        // also count them all!
-        $co = PrivateMessagePeer::instance()->selectCount($c);
-
-        $c->addOrderDescending("message_id");
+        $drafts_count = UserMessage::drafts($runData->user())->orderBy('id','DESC')->count();
```

Comment:
If we're just doing `COUNT(*)` then `ORDER BY` shouldn't matter at all right?

Reply:
Right, earlier in this code I was doing multiple things with this one var, I forgot to remove that clause. Fixed.